packages/compass-import-export/src/components/import-modal.tsx

```diff
@@ -168,12 +168,16 @@ function ImportModal({
   if (isOpen && !fileName && errors.length === 0) {
     // Show the file input when we don't have a file to import yet.
     return (
-      <ImportFileInput
-        autoOpen
-        onCancel={handleClose}
-        fileName={fileName}
-        selectImportFileName={selectImportFileName}
-      />
+      // Don't actually show it on the screen, just render it to trigger
+      // autoOpen
+      <div style={{ display: 'none' }}>
```

Comment:
Just for my understanding, what is the relationship to the rest of the changes here?

Reply:
Ah, yes, sorry, should've left a comment: this is an existing issue that I'm just fixing as a drive-by

It seems like import flow starts by autoshowing system file open dialog through rendering file input with autoOpen property. It actually renders it in DOM and the only thing preventing it from showing up is that system dialog is blocking. Very rarely (it's harder to reproduce on prod build that on dev build) it actually renders before system dialog is shown and you can see it:

![image](https://github.com/mongodb-js/compass/assets/5036933/bcbb9dfb-dc3a-4bf2-8f00-c3e23211c520)

Generally speaking it would probably be better not to render it and trigger this side effect from the same action that starts the import flow, but I'd take more refactoring to address that I'd like to avoid for now

Reply:
Oh yeah, that makes sense :) LGTM then!